examples/nginx/components/app/nginx.jsonnet

```diff
@@ -0,0 +1,54 @@
+local kap = import "lib/kapitan.libjsonnet";
+local inv = kap.inventory();
+
+local myContainers = [
```

Comment:
You can use `kube.libjsonnet` from the k8s example for `Container`, `Deployment` and `Service`.